changelog.d/7585.bugfix

```diff
@@ -0,0 +1 @@
+- Fix bug in automatic user creation on first time login with m.login.jwt. Regression in v1.6.0.
```

Comment:
The list markup gets added automatically by the tooling. I made a couple of suggestions below too:

```suggestion
Fix a bug in automatic user creation during first time login with `m.login.jwt`. Regression in v1.6.0.
```

You can also include "Contributed by <your name or handle>." if you'd like.

Reply:
Thanks for the review! I've removed the list markup and incorporated your suggestions.